src/moscot/plotting/_utils.py

```diff
@@ -161,22 +165,19 @@ def _sankey(
                     leftWidths[leftLabel]["bottom"] += dataFrame.loc[leftLabel, rightLabel]
                     rightWidths[rightLabel]["bottom"] += dataFrame.loc[leftLabel, rightLabel]
 
-                    if ind == 0:
-                        ax.fill_between(
-                            np.linspace(0 + left_pos[ind], xMax + left_pos[ind], len(ys_d)),
-                            ys_d,
-                            ys_u,
-                            alpha=0.65,
-                            color=colorDict[labelColor],
-                        )
+                    arr = np.linspace(0 + left_pos[ind], xMax + left_pos[ind], len(ys_d))
+                    color = (
```

Comment:
You use the `if interpolate_color` below, causes unnecessary repetition, would move the `color = ...` to each branch below.